spaces/S001103/properties/P000191.md

```diff
@@ -0,0 +1,7 @@
+---
+space: S001103
+property: P000191
```

Comment:
This property is hereditary: can we defer to a smaller power with a non G delta point?

Reply:
Ah, in that case I can add lack of P191 to the continuum power of countable discrete space (S101), for which P191 is currently unknown, and then just reference that for S1103. (Though, somewhat off-topic, do you know why the numbering is 1103 for this space?)

Reply:
https://github.com/pi-base/data/commit/77e6ee1fc93ab735f0d9cfc429daf09a9177d86a

It's space 103 in *Counterexamples* and long ago there was an attempt to match our IDs to that, though there was a mix up making everything usually a few off. But calling it S001103 was an attempt to keep the connection to 103.

Reply:
Yeah - I think it *was* S103, but somewhere our numbering got changed, removing the connections of pi-Base space IDs to IDs from *Counterexamples*. 😢